src/pages/Scan.vue

```diff
@@ -110,7 +110,7 @@ export default {
 
                             } else if (data.fail.length > 0) {
                                 this.showFail();
-                                this.setMessage("[xXx]It looks like this code has already been added, please double check and try again. If you are still unable to add the voucher code, don\'t worry - you will still receive payment if you send it in with your other vouchers.", constants.MESSAGE_WARNING);
```

Comment:
This is changing in duplicate message code https://github.com/neontribe/ARCVService/tree/1.1/271-better-errors - but fine to go in here like this.